runtime-config-linux.md

```diff
@@ -155,7 +155,7 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
    ]
 ```
 
-rlimits allow setting resource limits. The type is from the values defined in [the man page](http://man7.org/linux/man-pages/man2/setrlimit.2.html). The kernel enforces the soft limit for a resource while the hard limit acts as a ceiling for that value that could be set by an unprivileged process.
+rlimits allow setting resource limits. `type` is a string with a value from those defined in [the man page](http://man7.org/linux/man-pages/man2/setrlimit.2.html). The kernel enforces the `soft` limit for a resource while the `hard` limit acts as a ceiling for that value that could be set by an unprivileged process.
```

Comment:
Touching this line is probably a good enough excuse to reformat with one line per-sentence.